.goreleaser.yml

```diff
@@ -28,3 +28,19 @@ changelog:
     exclude:
     - '^docs:'
     - '^test:'
+brews:
```

Comment:
This will automatically build [this file](https://github.com/instructure-bridge/homebrew-truss-cli/blob/master/Formula/truss-cli.rb#L1) with using [this config](https://goreleaser.com/customization/homebrew/).